src/services/MovieService.js

```diff
@@ -0,0 +1,9 @@
+class MovieService {
+  getMovies() {
+    return fetch('http://api.tvmaze.com/shows')
+      .then((response) => response.json());
+  }
+}
+
+export { MovieService };
+export default new MovieService();
```

Comment:
Do you need those two exports here?

Reply:
Not really, at first i exported the class, then switched to an instance, but never cleaned up the class export